penguin/integrations/mcp/server_tools/ituv.py

```
@@ -0,0 +1,619 @@
+                        task_id,
+                        action,
+                        before,
+                        {"phase": TaskPhase.PENDING.value},
+                    )
+                project_manager.update_task_phase(task_id, TaskPhase.PENDING, reason=reason)
+            else:
+                return {
+                    "error": "unsupported_action",
+                    "supported_actions": ["set_status", "set_phase", "block", "unblock"],
+                }
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)